### PR TITLE
[JENKINS-32535] Improved test coverage 'add-job-to-view' and 'remove-job-from-view'

### PR DESCRIPTION
**Pull request by pjanouse**
Please review, thx Pavel

Note: I've needed to upgrade dependency for jenkins-test-harness (to version 2.6) because it incorporates a fix for JENKINS-32273 which I rely on.

### CODE REVIEW

File: test/src/test/java/hudson/cli/ViewManipulationTestBase.java [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+/**
+ * @author pjanouse
+ */
+public abstract class ViewManipulationTestBase {
+
+    protected CLICommandInvoker command;
+
+    @Rule public final JenkinsRule j = new JenkinsRule();
+
+    @Before abstract public void setUp();
```
Comment (olivergondza) [inline test/src/test/java/hudson/cli/ViewManipulationTestBase.java]: It is better design to have final `@Before` method in the parent calling abstract `getCommand`. The contract of `setUp` configuring `command` field is unclear this way.

**Comment by olivergondza** [comment]
:+1: with a minor comment.

**Comment by oleg-nenashev** [comment]
:+1:

**Comment by pjanouse** [comment]
Fixed as Oliver recommended.

**Pull request merged**